lblod/frontend-subsidiepunt
# update form dependencies

## ID

DGS-261

## Description

Update form dependencies needed to load 'lokaal bestuurlijk talent' subsidy.

## Type of change


## How to test

Run frontend with backend, create 'lokaal bestuurlijk talent' subsidy and verify it's working.
